### ISSUE TITLE
Migrate Photoview from Environment Variables configuration to a config file approach

### ISSUE THREAD

**Issue by kkovaletp** [photoview/photoview#1134]
**Is your feature request related to a problem? Please describe.**
Not a problem, just an inconvenience: we have many environmental variables and more to come. The current approach of `.env` file to manage them is not the most convenient way.

**Describe the solution you'd like**
It would be great to have a dedicated config file, formatted in YAML (preferably, as it is already used in `docker-compose`) or TOML, as it provides powerful features and visually clear and easily read structure.

**Additional context**
This topic was originally discussed in the https://github.com/photoview/photoview/pull/1130 PR.

**Comment by mafrosis** [comment]
FWIW I prefer ENV vars as it's one less file to consider.

All config for photoview is current in the `environment` block of my `docker-compose.yaml`. This simplicity is desirable for someone who runs many services in docker.

**Comment by kkovaletp** [comment]
Good point, @mafrosis. I'd agree that having a list of environment variables is easier for docker service management.
We would need to carefully review all the pros and cons before making the decision here...
